nipype/pipeline/engine/tests/test_engine.py

```diff
@@ -626,6 +626,38 @@ def func1(in1):
     assert not error_raised
 
 
+def test_parameterize_dirs_false(tmpdir):
+    from .... import config
+    from ....interfaces.utility import IdentityInterface
+    from ....testing import example_data
+
+    config.update_config({'execution': {'parameterize_dirs': False}})
+
+    wd = str(tmpdir)
+    os.chdir(wd)
```

Comment:
these two statements are not required for a workflow since you set `wf.base_dir = wd` later.